pkg/monitoring/fcdclient/client.go

```diff
@@ -0,0 +1,73 @@
+package fcdclient
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"go.uber.org/ratelimit"
+)
+
+func New(
+	fcdURL string,
+	requestsPerSec int,
+) Client {
+	return &client{
+		fcdURL,
+		&http.Client{},
+		ratelimit.New(int(requestsPerSec),
+			ratelimit.Per(1*time.Second), // one request every 1 second
+			ratelimit.WithoutSlack,       // don't accumulate previously "unspent" requests for future bursts
+		),
+	}
+}
+
+type client struct {
+	fcdURL     string
+	httpClient *http.Client
+	limiter    ratelimit.Limiter
+}
+
+func (c *client) GetTxList(ctx context.Context, params GetTxListParams) (Response, error) {
+	_ = c.limiter.Take()
+	query := url.Values{}
+	if params.Account.String() != "" {
+		query.Set("account", params.Account.String())
```

Comment:
should be Add's not Set's

Reply:
At this point setting multiple params is not needed. If/When we do batching, I'll add it!